src/plugins/workspace/public/components/service_card/workspace_list_card.tsx

```diff
@@ -89,104 +88,118 @@ export const WorkspaceListCard = (props: WorkspaceListCardProps) => {
 
   return (
     <EuiPanel paddingSize="s" hasBorder={false} hasShadow={false}>
-      <EuiFlexGroup gutterSize="xs" alignItems="center">
-        <EuiFlexItem grow={3}>
-          <EuiTitle>
-            <h4>Workspaces</h4>
-          </EuiTitle>
+      <EuiFlexGroup
+        direction="column"
+        justifyContent="spaceBetween"
+        style={{ height: '100%' }}
+        gutterSize="none"
+      >
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup gutterSize="xs" alignItems="center">
+            <EuiFlexItem grow={3}>
+              <EuiTitle>
+                <h4>Workspaces</h4>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={1}>
+              <EuiToolTip position="bottom" content={WORKSPACE_LIST_CARD_DESCRIPTION}>
+                <EuiIcon type="iInCircle" aria-label="workspace list card description" />
+              </EuiToolTip>
+            </EuiFlexItem>
+            <EuiFlexItem grow={5}>
+              <EuiSelect
+                value={filter}
+                data-test-subj="workspace_filter"
+                onChange={(e) => {
+                  setFilter(e.target.value);
+                }}
+                options={[
+                  {
+                    value: 'viewed',
+                    text: i18n.translate('workspace.list.card.filter.viewed', {
+                      defaultMessage: 'Recently viewed',
+                    }),
+                  },
+                  {
+                    value: 'updated',
+                    text: i18n.translate('workspace.list.card.filter.updated', {
+                      defaultMessage: 'Recently updated',
+                    }),
+                  },
+                ]}
+              />
+            </EuiFlexItem>
+            {isDashboardAdmin && (
+              <EuiFlexItem grow={false}>
+                <EuiToolTip position="top" content="Create workspace">
+                  <EuiButtonIcon
+                    data-test-subj="create_workspace"
+                    aria-label="create workspace"
+                    display="base"
+                    iconType="plus"
+                    size="m"
+                    onClick={() => {
+                      application.navigateToApp(WORKSPACE_CREATE_APP_ID);
+                    }}
+                  />
+                </EuiToolTip>
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
         </EuiFlexItem>
-        <EuiFlexItem grow={1}>
-          <EuiToolTip position="bottom" content={WORKSPACE_LIST_CARD_DESCRIPTION}>
-            <EuiIcon type="iInCircle" aria-label="workspace list card description" />
-          </EuiToolTip>
+        <EuiSpacer />
+        <EuiFlexItem grow={true}>
+          {workspaceList && workspaceList.length === 0 ? (
```

Comment:
This seems is existing code, but would be nice to change it in this way, as empty state should be displayed if `workspaceList` is undefined.
```suggestion
          {!workspaceList || workspaceList.length === 0 ? (
```

Reply:
Okok, Thanks for your comments~